unified-codes/tools/scripts/dgraph/data/v2/schema.gql

```diff
@@ -0,0 +1,83 @@
+type Property {
+  type
+  value
+}
+
+type Category {
+  code
+  name
+  children
+  properties
+}
+
+type Product {
+  code
+  name
+  combines
+  children
+  properties
+}
+
+type Route {
+  code
+  name
+  children
+  properties
+}
+
+type DoseForm {
```

Comment:
```
export interface IFormEntity extends IEntity {
  type: EEntityType.Form;
  children?: IFormEntityChildren;
  properties?: IFormEntityProperties;
}
```

Type is currently just `FormEntity`, suggest changing it to `DoseFormEntity`?

Reply:
Yep yep, can do!